templates/mia-default/settings.yaml

```diff
@@ -0,0 +1,95 @@
+
+# General settings.
+general:
+  # The CyanogenMod codename of the device.
+  cm_device_codename:
+  # Preferred CyanogenMod release type
+  cm_release_type: snapshot
+  # Preferred CyanogenMod release version
+  cm_release_version: M12
+
+
+# Mapping of the items in the definition and their place in the update archive.
+update_content:
+  - src: init.d
+    dst: system/etc/init.d
+  - src: META-INF
+    dst: META-INF
+  - src: misc
+    dst: sdcard/misc
+  - src: system-apps
+    dst: system/priv-app
+  - src: system-lib
+    dst: system/lib
+  - src: user-apps
+    dst: data/app
+  - src: user-data
+    dst: data/data
+  - src: other/update-binary
+    dst: META-INF/com/google/android/update-binary
+  - src: other/updater-script
+    dst: META-INF/com/google/android/updater-script
+  - src: other/firstboot.sh
+    dst: data/.firstboot.sh
+
+
+# Apps with direct download link.
+other_apps:
+  - name: F-Droid
+    url: https://f-droid.org/repo/org.fdroid.fdroid_810.apk
```

Comment:
This needs to go in system apps directory in order for fdroid to work with "Unknown Sources" being allowed (which is insecure).

So there are a few types of apps: system apps (just fdroid right now, but could be others), fdroid apps, and direct download apps (with just a link, like orfox-debug). Perhaps the best approach is to have just an apps key, and there are two optional keys for each: `install_type` (system, user, or privileges -- each has different install dirs) and `source_type` (fdroid or direct download, for now).

Thoughts? (we can do this later, just noticed while comparing the generated zips with my last working one)